doc/whats_new.rst

```diff
@@ -28,6 +28,21 @@ Fixes
 Contributors
 ------------
 
+The following people contributed to this release (in alphabetical order)
+
+	Ana Luisa Pinho
+	Anthony Gifuni
+	Bertrand Thirion
+	Chris Markiewicz
```

Comment:
Duplicate

```suggestion
```

Reply:
What's your commit email?

Reply:
Think I found it. Tha stanford one.

Reply:
Yeah, mostly Stanford. Some GitHub-generated commits will show up on gmail. These two lines will generally work:

https://github.com/nipy/nipype/blob/536c57da150d157dcb5c121af43aaeab71cdbd5f/.mailmap#L34-L35